sdk/data/azcosmos/cosmos_example_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
```

Comment:
Based on other examples: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/example_test.go, https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/data/aztables/example_test.go#L4, name the file `example_test.go` and change the package to `azcosmos_test`